### PR TITLE
[Jenkins] Ignore some warnings in Docs job.

### PR DESCRIPTION
**Pull request by bilke**
[12 warnings left ...](https://jenkins.opengeosys.org/job/OGS-6/job/bilke/job/doxygen-parser/5/warnings15Result/)

There were no warnings concerning max graph ...

### CODE REVIEW

**Comment by endJunction** [comment]
I'll look into the remaining warnings.

**Comment by endJunction** [comment]
@bilke Could it be that perl is not installed on dockers?
Problems running bibtex. Verify that the command 'perl --version' works from the command line. Exit code: 2

**Comment by endJunction** [comment]
The `cite` warnings are due to some errors in Doxygen. The bibliography generation works fine with 1.8.12 version.

The known problem of wrong link generation for cite is _not_ fixed in 1.8.11.

**Comment by endJunction** [comment]
@rinkk Could you please add documentation to 'window' parameter here:
DiagramPrefsDialog.h:69

The following parameters of DiagramPrefsDialog::DiagramPrefsDialog(const QString &filename, DetailWindow *window=NULL, QDialog *parent=0) are not documented: parameter 'window'

I could not guess the meaning of that parameter.

File: scripts/jenkins/docs.groovy [hunk 1]
```diff
@@ -25,6 +25,7 @@ stage('Reports (Docs)') {
         keepAll: false, reportDir: 'build/docs', reportFiles: 'index.html',
         reportName: 'Doxygen'])
     step([$class: 'WarningsPublisher', canResolveRelativePaths: false,
+        messagesPattern: '.*ogs_file_.*,.*potential recursive class relation.*',
```
Comment (endJunction) [inline scripts/jenkins/docs.groovy]: They will become active end of this week, when ogs_file_param are all documented.

**Comment by bilke** [comment]
Done: Zero doxygen warnings in Jenkins.

**Pull request merged**